lib/platform.js

```diff
@@ -3,14 +3,15 @@ const { extname } = require('path')
 
 module.exports = fileName => {
   const extension = extname(fileName).slice(1)
+  const arch = fileName.includes('arm64') | fileName.includes('aarch64') ? '_arm64' : ''
```

Comment:
Let's make this a bit easier to understand, IMO:

```suggestion
  const arch = (fileName.includes('arm64') || fileName.includes('aarch64')) ? '_arm64' : ''
```